src/blog_vi/translations/providers/deepl.py

```diff
@@ -1,13 +1,13 @@
 from .base import BaseTranslateProvider
 import deepl
-from blog_vi._config import DEEPL_API_KEY
 
 
 class DeeplTranslateProvider(BaseTranslateProvider):
     id = 'deepl'
+    settings_key = 'deepl_translator'
 
     def translate(self, text: str, source_abbreviation: str, target_abbreviation: str) -> str:
-        translator = deepl.Translator(DEEPL_API_KEY)
+        api_key = getattr(self, '_BaseTranslateProvider__api_key')
+        translator = deepl.Translator(auth_key=api_key)
```

Comment:
remove this, you have a `self.translator`

Reply:
This is just instance of deepl.Translator() class, not identifying which translato